tests/test_config.py

```diff
@@ -282,6 +282,44 @@ def test_load_config_safe_external(self) -> None:
         SCUBA_YML.write_text(f"image: !from_yaml {external_yml} danger")
         self.__test_load_config_safe(external_yml)
 
+    def test_load_config_image_name(self) -> None:
+        GITLAB_YML.write_text(
+            """
+            two:
+              stage: build
+              image:
+                name: dummian:8.2
```

Comment:
It is not obvious what exactly is being tested here. I had to read it several times to figure it out.

A comment like this would go a long way in aiding understanding.

```python
# .gitlab-ci.yml supports a long-form `image` specification
# with `name` and other sub-keys:
# https://docs.gitlab.com/ee/ci/yaml/#image
# 
# Scuba doesn't currently support that, so make sure we handle
# this error gracefully.
#
# TODO(#230): Support this complex `image` format.
```